app/src/crypto.c

```diff
@@ -745,14 +810,15 @@ typedef struct {
 } tmp_checkoutput;
 
 // handleCheckandSign step 7/11
-zxerr_t crypto_checkoutput_sapling(uint8_t *buffer, uint16_t bufferLen, const uint8_t *txdata, const uint16_t txdatalen) {
+zxerr_t crypto_checkoutput_sapling(uint8_t *buffer, uint16_t bufferLen, const uint8_t *txdata, const uint16_t txdatalen, const uint8_t tx_version) {
+    zemu_log_stack("crypto_checkoutput_sapling");
     MEMZERO(buffer, bufferLen);
 
     if(get_state() != STATE_CHECKING_ALL_TXDATA){
         return zxerr_unknown;
     }
-
-    if(length_t_in_data() + length_spenddata() + length_outputdata() + LENGTH_HASH_DATA != txdatalen){
+    //TODO: Probably need to change for v5 tx checks
```

Comment:
Still a TODO?

Reply:
Yes, I forgot about it because they are both the same length (when there are no orchard inputs), but it will not be the case in future when we add support for Orchard, I'll fix that now.